### PR TITLE
Changed tutorial to academy in header, in FR & EN

### PR DESCRIPTION
**Pull request by etienneburdet**
Replace tutorial link with academy. Works in EN/FR

### CODE REVIEW

File: source/_templates/header.html [hunk 1]
```diff
@@ -41,8 +41,8 @@
             <a class="ods__documentation-header-nav-item-active" href="{{ pathto(master_doc) }}">FAQ</a>
             <a href="/widgets/">Widgets</a>
             <a href="/{{ language }}/apis/">APIs</a>
-            <a href="/tutorials/{{ language }}/">{{ gettext('Tutorials') }}</a>
+            <a href="/tutorials/{{ language }}/">Academy</a>
```
Comment (richterb) [inline source/_templates/header.html]: I think this should be a link to `{{ gettext('https://academy.opendatasoft.com/page/homepage') }}` as well

Comment (etienneburdet) [reply source/_templates/header.html]: Yep, it's what I have in local… :p

**Review by Amli — CHANGES_REQUESTED** [review]
The translations should be modified in transifex otherwise they will be overridden on next translations update.
Also I'm not sure we want to actually "translate" the url to Academy, we need to find another way to handle the 2 different urls.

**Comment by etienneburdet** [comment]
Should be all good. I don't get the Travis error, the exact command works fine on local :/

**Pull request merged**